main.css

```diff
@@ -0,0 +1,97 @@
+.heading {
+    font-family: 'Apple-Chancery', cursive;
```

Comment:
this font will probably only work on your machine - you can import Google fonts to ensure that the font is loaded for all users https://fonts.google.com/